driver/default_prom.c

```diff
@@ -0,0 +1,10 @@
+const size_t default_prom_nentries = 3;
```

Comment:
Would be nice to autogenerate this from an appropriate configuration file instead.

Reply:
Done, adding auto-generation has been a bit tricky, I had to add it to Kbuild in order to also consider it for dkms